qemu-libvirt/cl/machine-mynode.yaml.tmpl

```diff
@@ -1,4 +1,6 @@
 ---
+variant: flatcar
+version: 1.1.0
```

Comment:
This migrates it from CLC to Butane, thanks. For that to be in sync with the rest we need to bump the `version` of the ct provider

Reply:
and remove `filesystem: root` below

Reply:
I also bumped the libvirt provider to use the 0.7.x versions

Reply:
Thanks